docker-compose.yml

```diff
@@ -12,6 +12,12 @@ services:
       - "8080:8080"
     environment:
       - PYTHONUNBUFFERED=1
+      - ENGINE=${ENGINE}
+      - NAME=${NAME}
+      - USER=${USER}
```

Comment:
Using `USER` as an environment variable name might cause conflicts since `USER` is a reserved environment variable in many systems. Consider renaming it to something more specific, like `DB_USER`.